# Meeting notes from the first video call with Professor Camps

(0:00)
Majority of mutations you find will likely have nothing to do with resistance because the mutation rate is so high in HIV.
In E. coli drug resistant genes, you will expect most mutations taht you find are going to have somethign to do with resistance, but it's nto the case for HIV.
You find all kinds of mutations in HIV, not just those due to drug resistance.
Need to focus on spectrum as a signature for the polymerase rather than the mutations as sources for mutations.

(1:00)
Profile of 12 possible bp substituions, but double-stranded stage of virus brings that down to 6 possible bp substitutions.
Look at percent of each of the 6 changes gives a rough profile.
More recently, people have been looking at two neighboring nulceotides, not just those change.
This gives you a more high resolution profile.
With the numbers we have, going to that 96 may be reaching. Ie we don't have enough mutations.

(2:10)
Kinetics of structure with PyRosetta.
That alone won't go far.
If you find a mutaiton that seems to alter the spectruem and the structure suggest is critival for discrimination, then it's useful for us.

(2:50)
What parts are critical for functionality?
Given by other papers.  Studies of polymerases.  Find the papers given in his proposal document.

(3:25)
Six pais of mutation types give by figure with black/white columns.

(4:00)
Figure 2 shows which areas of the polymerase are functionally important.

(4:10)
Not every mutation, just every mutation that is KNOWN to be a drug-resistant mutations.
Well-known positions given in a popular paper.

(5:00)
Figure 2 shows drug-resistance mutations.
Compare indexes to compare how different the profile of this mutatant is to the total.
Higher bar --> more likely ot have a differnt profile.

(5:30)
Higher bar --> more different the spectrum was.

(5:35)
Figure 3.
Black --> spectrum from one mutant.
White --> spectrum from another mutant.
They are different.
This is captured in the comparison index from Fig 2.
Difference b/w mutant and rest of pop. fo rthe nuc profile.

(6:15)
Figure 3 clarifications.
Changes from before and after treatment, or more generally, time 1 and time 2.
Some patients not treated.
Shows how the virus changes even w/o treatment.

(7:00)
Resistance caused by individual mutations.
K70R is known to cuase resistance.
Does the profile of nuc sub look diff from sequences that have that mutation or not.
__Is K70R a single point substitution?__
Figure 3.

(7:50)
AA substitutions - codon changing to a diff codon at a given position.
Nuc substituions, regardless of codon effects.
Here, it's the same target.
Mutaant and sequence that we're looking for mutations in is the same.

(9:00)
E. coli example to further explain why this recursion is not the typical look at things.

(9:50)
In HIV, it's a much smaller genome.
The sequences that we have are all from the polymerase itself.
We're looking at for a given mutant polymerase, such as K70R, we're looking the profile of random mutation that happen in the genome, but the genome is the polymerase itself.

(10:40)
Fig 3.
For any given sample, the total number of nucs taht are diff from the last pt and the first pt.
Do this rather than having a reference sequence.
Samples taken over years, virus has chnaged.
Dont' want old mutations.
Large samples might be past genetic variation.

(11:45)
Power of analysis is looking for independent events.
This is the ultaimate way of looking at independent events.
Able to see in different patients if the had the same change, it was still independent.

(12:20)
Figure 3.
Black vs white, are these differences significant?
If sample size is 50 mutations, a 5% diff from wildtype and K70R might not be significant.
A 10% form black to white might not be different at all either.
__How do we quantify/establish significance differences for a given RT mutant?__
Start with mutants that are most significant(frequent?) in order to retain the highest statistical power.

(13:45)
Look at the most frequency.
Statistical method for this is our own project to look at.

(14:30)
Rank the RT mutatns by the degree of significance.
Then go to 3D structure and map them.

(14:45)
Nuc analogs and non-nuc analogs.
Anything that gives resistance to a nuc analog is going to be important for fidelity by definition.
That class of drugs doesn't have much descriminating power for this bc it's expected.
Mutant happens to have a diff spectrum and happens to be at the active site, it's expected.

(15:40)
NNRTI drugs don't target active site, target diff place in protein and alter confirmation of protein.
If drug resistant has impact on fidelity and is located near active site, that is more telling, 
bc it is not as expected.

(16:45)
If when you select for resistance in HIV, if you are at the same time changes the varianc(virulence?) eof virus.
Ability fo the virus to generate geneti diversity is essential for virulence.

(17:10)
NRTI target active site of protein, so they affect virulence, don't know if it's increasing or decreasing virulence.
Too unstable, wipe outs virus.
Too stable, immune system wipes out virus.
All we know really is there is modulation.

(18:20)
__Impact of mutations as a result of treatment to the drugs.__

(18:30)
Mutations occur more frequently from different drugs.
Known ones are given by paper...

Down time...

(21:20)
Structure --> paper number 5, Fianos

Down time...

(23:00)
9 and 10 on the paper.
These are good for list of mutations in RT that are known to cause drug resistance.

(24:20)
Fidelity is determined how the enzyme opens and closes.
Closed --> favors incorportation into masent strand, favors mistakes, confers nucleotides w/o checks.

(25:10)
Different part of the project here if time permits.

(25:50)
Another aspect to look at is that the number of mutations varies dramatically between patients.
How much time elapsed between t1 and t2? Not a lot of info.
Mutatn with diff profile if that actually mathces taht have more mutations overall.

(26:40)
Look at spectra as a way of not having the look at frequency.
Bc we don't have a good estimate of replication cycles between t1 and t2 
w/o a way to tell if there is a difference is to look at whether the changes are different.

(27:10)
Once we have the mutants that have an affect on fidelity, good to look at frequency.


(27:30)
Parts of virus are ss, some are ds, in the RNA form.
The part that are dsRNA have a hit selection to maintain the base pair.
You have a G-->A mutation on one strand, have a huge selection to have a C-->A selection on other strand (is given or more likely?)

(28:00)
If you have enough mutations, subtract all of the mutations that fall into the ds area.
ds mutations may be due to rare occurance needed to maintain structure of virus.

(28:50)
Analysis to see differences.
How do we statistically prove it?

(29:10)
If very mutations selected by drugs affect virulility...
Come back here

Diagnostic vs treatment.
3-4 drugs at same time.
Given as a cocktail.

Still have to use other drugs in mixture.
Hard to tell actual effect of evolutionary trajectory of the virus.
Immediate effect is diagnostic.








